AY2021S2-CS2103-T14-1/tp
# Fix #257: Unbold example commands in quick start

## What this does
Fixes #257

## How to test
1. cd to `docs/` folder of the project.
2. Run `bundle exec jekyll serve`.
3. Open the jekyll-hosted website in your browser (default address: http://127.0.0.1:4000).
4. Check that the commands are not bold in the UG Quick Start section.
